### PR TITLE
Fix some of the manual help text

### PR DESCRIPTION
**Pull request by thockin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,11 +180,12 @@ OPTIONS
...
             Within quoted values, commas MAY be escaped, but are not required
-            to be.  Any other escape sequence is an error.  (default: "")
+            to be.  Any other escape sequence is an error.
 
     --git-gc <string>, $GIT_SYNC_GIT_GC
-            The git garbage collection behavior: one of 'auto', 'always',
-            'aggressive', or 'off'.  (default: auto)
+            The git garbage collection behavior: one of "auto", "always",
+            "aggressive", or "off".  If not specified, this defaults to
+            "auto").
```
Comment (stp-ip) [inline README.md:188]:
```suggestion
            "auto".
```

File: cmd/git-sync/main.go [hunk 2]
```diff
@@ -1877,11 +1878,12 @@ OPTIONS
...
             Within quoted values, commas MAY be escaped, but are not required
-            to be.  Any other escape sequence is an error.  (default: "")
+            to be.  Any other escape sequence is an error.
 
     --git-gc <string>, $GIT_SYNC_GIT_GC
-            The git garbage collection behavior: one of 'auto', 'always',
-            'aggressive', or 'off'.  (default: auto)
+            The git garbage collection behavior: one of "auto", "always",
+            "aggressive", or "off".  If not specified, this defaults to
+            "auto").
```
Comment (stp-ip) [inline cmd/git-sync/main.go:1886]:
```suggestion
            "auto".
```

**Review by stp-ip — APPROVED** [review]
/approve with the 2 suggestions.

File: README.md (hunk 1 above)
Comment (thockin) [reply README.md:188]: Fixed in next push

File: cmd/git-sync/main.go (hunk 2 above)
Comment (thockin) [reply cmd/git-sync/main.go:1886]: Fixed in next push

**Comment by thockin** [comment]
Re-pushed

**Review by stp-ip — APPROVED** [review]
/lgtm

**Pull request merged**